src/view/modals/TransferFactory.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {
+  DialogTitle,
+  Dialog,
+  DialogContent,
+  DialogContentText,
+  IconButton,
+  Grid,
+} from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
```

Comment:
In general, I have been using named imports/destructuring.

```suggestion
import { Close } from '@material-ui/icons';
```

[More info](https://material-ui.com/components/icons/#usage) on pros/cons of each approach if you're curious.